ui/public/assets/images/vector.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="4" viewBox="0 0 12 4" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please choose a better name for this image. `vector.svg` reveals nothing about its conents.